### ISSUE TITLE
SftpClient.stat group status Error

### ISSUE THREAD

**Issue by echoescn** [jyu213/ssh2-sftp-client#121]
https://github.com/jyu213/ssh2-sftp-client/blob/master/src/index.js#L122
I should write it like this
```
sftp.stat(remotePath, function (err, stats) {
      if (err){
        reject(new Error(`Failed to stat ${remotePath}: ${err.message}`));
      } else {
      // format similarly to sftp.list
        console.log(stats);
        resolve({
          mode: stats.mode,
          permissions: stats.permissions,
          owner: stats.uid,
          group: stats.gid,
          size: stats.size,
          accessTime: stats.atime * 1000,
          modifyTime: stats.mtime * 1000
        });
      }
    });

```

`stats.guid` change `stats.gid`

**Comment by theophilusx** [comment]
thanks for catching that. Have added fix to my fork which will be pulled across once I do a PR

**Comment by jyu213** [comment]
published version 2.5.0
